test/functional_tests/README.md

```diff
@@ -0,0 +1,25 @@
+Functional Tests
+================
+
+How to run functional tests?
+
+* Edit /etc/ansible/hosts and add at least one machine to the group called
+  clients. E.g.:
+
+```ini
+[clients]
+centos7.example.com
+```
+
+* You have to be able to login to all machines in group using ssh key
+  (not password).
+* Execute following commands:
+
+```bash
+cd ./ansible_playbooks
+ansible-playbook ./configure_package_manager.yml --extra-vars="candlepin_hostname=<IP_OF_TESTING_CANDLEPIN>"
```

Comment:
We have configured the IP of the host to 192.168.111.1 in the Vagrantfile. Perhaps just make that the default on this setup.